src/components/Header/index.js

```diff
@@ -73,6 +73,10 @@ export default ({
     setActiveDatasetManager(newDM)
   })
 
+  const onJoinSession = useEventCallback(async (sessionName) => {
+    window.location.href=sessionName
+  })
```

Comment:
hmm, we should have to trigger a page reload to do this. I think what would be better is if we used `setActriveDatasetManager` (it should be really similar to what we do in the [use-enter-collaborative-session file](https://github.com/UniversalDataTool/universal-data-tool/blob/master/src/hooks/use-enter-collaborative-session/index.js))